src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import Button from './Button';
+
+function ButtonPanel() {
```

Comment:
## Enforce the use of `ES6` syntax
- [ ] Kindly change this function declaration into an arrow function.